Translations/uk.po

```diff
@@ -5387,12 +5383,12 @@ msgstr "Бронзокулачний Вогнекамінь"
 #: Source/monstdat.cpp:524
 msgctxt "monster"
 msgid "Wrathfire the Doomed"
-msgstr ""
+msgstr "Приречений Гнівовогонь"
 
 #: Source/monstdat.cpp:525
 msgctxt "monster"
 msgid "Firewound the Grim"
-msgstr ""
+msgstr "Нежадний Вогнеран"
```

Comment:
Нежадний not greedy? I don't think this means Grim. 

What do you think about Похмурий Вогнеран ?

Reply:
I meant Нещадний -_-

Reply:
Нещадний is like....  Merciless?

Reply:
It can be translated like that: https://e2u.org.ua/s?w=grim&dicts=all&highlight=on&filter_lines=on
And considering it's a monster name, just Похмурий is not fierce enough

Reply:
yeah I think you are right. Похмурий  seems somewhat sad . 

I am thinking though, суворий is harder and might fit better, but I am not certain it is the best word.